soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -10215,25 +10216,24 @@ void func_80848B44(GlobalContext* globalCtx, Player* this) {
     Vec3f shockPos;
     Vec3f* randBodyPart;
     s32 shockScale;
+        this->shockTimer--;
+        this->unk_892 += this->shockTimer;
 
-    this->shockTimer--;
-    this->unk_892 += this->shockTimer;
+        if (this->unk_892 > 20) {
+            shockScale = this->shockTimer * 2;
+            this->unk_892 -= 20;
 
-    if (this->unk_892 > 20) {
-        shockScale = this->shockTimer * 2;
-        this->unk_892 -= 20;
-
-        if (shockScale > 40) {
-            shockScale = 40;
-        }
+            if (shockScale > 40) {
+                shockScale = 40;
+            }
 
-        randBodyPart = this->bodyPartsPos + (s32)Rand_ZeroFloat(ARRAY_COUNT(this->bodyPartsPos) - 0.1f);
-        shockPos.x = (Rand_CenteredFloat(5.0f) + randBodyPart->x) - this->actor.world.pos.x;
-        shockPos.y = (Rand_CenteredFloat(5.0f) + randBodyPart->y) - this->actor.world.pos.y;
-        shockPos.z = (Rand_CenteredFloat(5.0f) + randBodyPart->z) - this->actor.world.pos.z;
+            randBodyPart = this->bodyPartsPos + (s32)Rand_ZeroFloat(ARRAY_COUNT(this->bodyPartsPos) - 0.1f);
+            shockPos.x = (Rand_CenteredFloat(5.0f) + randBodyPart->x) - this->actor.world.pos.x;
+            shockPos.y = (Rand_CenteredFloat(5.0f) + randBodyPart->y) - this->actor.world.pos.y;
+            shockPos.z = (Rand_CenteredFloat(5.0f) + randBodyPart->z) - this->actor.world.pos.z;
 
-        EffectSsFhgFlash_SpawnShock(globalCtx, &this->actor, &shockPos, shockScale, FHGFLASH_SHOCK_PLAYER);
-        func_8002F8F0(&this->actor, NA_SE_PL_SPARK - SFX_FLAG);
+            EffectSsFhgFlash_SpawnShock(globalCtx, &this->actor, &shockPos, shockScale, FHGFLASH_SHOCK_PLAYER);
+            func_8002F8F0(&this->actor, NA_SE_PL_SPARK - SFX_FLAG);
```

Comment:
was any of this chanaged? in the diff it looks like just some of the indentation was modified.